cybered/hashing/templates/hashing/keyed_hashes_results.html

```diff
@@ -0,0 +1,75 @@
+<!doctype HTML>
+<html>
+{% include "includes/head.html" %}
+
+<body>
+    {% include "includes/header.html" %}
+    <div class="container">
+        <h3>Keyed Hashes</h3>
+
+        <p>
+            Below, you can see how we combined the text you entered with the password you entered.
+            We calculated the message digest for the text both with and without the password;
+            as you can see, adding the key changes the digest a lot.
+        </p>
+
+        <h4>Computed Message Digests</h4>
+        <p>The following table lists the computed message digests for the text you entered:</p>
+
+        <div class="card">
+            <div class="card-body">
+                <h5 class="card-title">Original Text</h5>
+                <div style="max-height: 100px; overflow-y: auto;">
+                    {{ input_text }}
+                </div>
+            </div>
+            <div class="card-body">
+                <h5 class="card-title">Password</h5>
+                <div style="max-height: 100px; overflow-y: auto;">
+                    {{ input_key }}
+                </div>
+            </div>
+            <div class="card-body">
+                <h5 class="card-title">Combined Text and Password</h5>
+                <div style="max-height: 100px; overflow-y: auto;">
+                    {{ keyed_text }}
```

Comment:
I'm not sure it matters, but somewhere any trailing spaces on the message are being discarded. It's possible that trailing space is stripped before the form data is POSTed, but that's just a guess.

Reply:
Yeah, I noticed that. I'm guessing it's on the django side where they 'clean' the post data. I don't think it's a problem